scss/_patterns_image.scss

```diff
@@ -0,0 +1,18 @@
+@import 'settings';
+
+@mixin vf-p-image {
+  .p-image {
+
+    &--bordered {
+      border: {
+        color: $color-mid-light;
+        style: solid;
+        width: 1px;
+      }
+    }
+
+    &--shadowed {
+      box-shadow: 0 1px 5px 1px rgba(51, 51, 51, .2); // sass-lint:disable-line no-color-literals
```

Comment:
You should use transparentize here http://sass-lang.com/documentation/Sass/Script/Functions.html#transparentize-instance_method here